main/http.c

```diff
@@ -1665,6 +1665,46 @@ struct ast_http_auth *ast_http_get_auth(struct ast_variable *headers)
 	return NULL;
 }
 
+struct ast_variable *ast_http_create_basic_auth_header(const char *userid,
+	const char *password)
+{
+	int encoded_size = 0;
+	int userinfo_len = 0;
+	RAII_VAR(char *, userinfo, NULL, ast_free);
+	char *encoded_userinfo = NULL;
+	struct ast_variable *auth_header = NULL;
+
+	if (ast_strlen_zero(userid)) {
+		ast_log(LOG_ERROR, "Invalid userid\n");
```

Comment:
I don't think from a user perspective this is really... useful. There's no details to go along with it.

Reply:
Will remove.